modules/nodes-cluster-resource-congifure.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * logging/nodes-cluster-overcommit.adoc
+
+[id="nodes-cluster-resource-congifure_{context}"]
+= Configuring cluster-level overcommit 
+
+
+The Cluster Resource Override Operator requires a `ClusterResourceOverride` custom resource (CR) 
+and a label for each project where you want the operator to control overcommit.
+
+
+.Prerequisites
+
+The Cluster Resource Override Operator overrides have no effect if limits have not
+been set on containers. Create a LimitRange object with default limits per individual project 
+or configure limits in Pod specs in order to for the overrides to apply.
+
+.Procedure 
+
+. Edit the `ClusterResourceOverride` CR:
++
+----
+apiVersion: operator.autoscaling.openshift.io/v1
+kind: ClusterResourceOverride
+metadata:
+-   name: cluster
+spec:
+   memoryRequestToLimitPercent: 25 <1>
+   cpuRequestToLimitPercent: 25 <2>
+   limitCPUToMemoryPercent: 200 <3>
+----
+<1> Optional. If a container memory limit has been specified or defaulted, specify the percentage to override the limit, between 1-100.
+<2> Optional. If a container CPU limit has been specified or defaulted, specify the percentage to override the limit, between 1-100.
+<3> Optional. If a container memory limit has been specified or defaulted, specify the percentage to override the limit, with a 100 percentage scaling 1Gi of RAM to equal 1 CPU core. This is processed prior to overriding CPU request (if configured).
+
```

Comment:
@mburke5678 can we also mention what are the default values for these 3 parameters if user does not specify a value?

Reply:
Added defaults.